launch_chffrplus.sh

```diff
@@ -6,6 +6,10 @@ export NUMEXPR_NUM_THREADS=1
 export OPENBLAS_NUM_THREADS=1
 export VECLIB_MAXIMUM_THREADS=1
 
+if [ -z "$NEOS_VERSION" ]; then
+  export NEOS_VERSION="15-RC2"
```

Comment:
Keep 14 here, and set 15-rc2 for in launch.sh for internal use

Reply:
Will do.